netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -195,8 +193,13 @@ private void onHeadersRead(ChannelHandlerContext ctx, int streamId, Http2Headers
           checkNotNull(transportListener.methodDetermined(method, metadata), "statsTraceCtx");
       NettyServerStream.TransportState state = new NettyServerStream.TransportState(
           this, http2Stream, maxMessageSize, statsTraceCtx);
-      NettyServerStream stream = new NettyServerStream(ctx.channel(), state, attributes,
+      Attributes attributesWithAuthority = Attributes
+              .newBuilder()
```

Comment:
This file is not particularly good about following the rule, but line continuation should be +4.